extensions/wikia/CommunityPage/CommunityPageSpecialController.class.php

```diff
@@ -410,4 +452,20 @@ private function formatTotalEditorsNumber( $editors ) {
 
 		return $editors;
 	}
+
+	/**
+	 * @param $userId
+	 * @return string name of DS svg file with proper badge
+	 */
+	private function getUserBadge( $userId ) {
```

Comment:
Did you consider moving this function to Design System extensions for example to `DesignSystemHelper` ?